cfme/infrastructure/cluster.py

```diff
@@ -132,6 +133,43 @@ def is_displayed(self):
             super(TimelinesView, self).is_displayed)
 
 
+class ClusterCollection(Navigatable):
```

Comment:
@quarckster We cannot add clusters from UI so create method will not exists.
Have added the instantiate method.